File: Benchmarks/benchmark.cpp
#include <iostream>
#include <chrono>

int fib(int n)
{
    if (n <= 2)
    {
        return 1;
    }
    else
    {
        return fib(n - 1) + fib(n - 2);
    }
}

int main()
{
    std::chrono::steady_clock::time_point begin = std::chrono::steady_clock::now();
    for (int i = 1; i < 41; i++)
    {
        fib(i);
    }
    std::chrono::steady_clock::time_point end = std::chrono::steady_clock::now();
    std::cout << "Time difference = " << std::chrono::duration_cast<std::chrono::milliseconds>(end - begin).count() << "[ms]" << std::endl;
}

File: Benchmarks/benchmark.r
fib <- function(n) {
   if (n <= 2) {
      return(1)
   } else {
      return(fib(n - 1) + fib(n - 2))
   }
}

ptm <- proc.time()

for (i in 1:40) {
   fib(i)
   print(i)
}

print(proc.time() - ptm)